pkg/trace/pb/appsec.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package pb;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+// AppSecStruct is a container for AppSec data sent by the tracers.
+message AppSecStruct {
+    repeated AppSecTrigger triggers = 1 [(gogoproto.jsontag) = "triggers", (gogoproto.moretags) = "msg:\"triggers\""];
+}
+
+// AppSecTrigger associates an AppSec rule and the data that triggered.
```

Comment:
The documentation in this file is a bit of a dead end. Can you please explain what these structures are in clear detail, or at least introduce a link where a reader could better understand them?

Reply:
I added a link to the library that is creating those triggers: https://github.com/DataDog/libddwaf. I don't think we have more public documentation for now. This proto file comes from our backend code.

Reply:
It's fine, it doesn't matter. We don't need public documentation. Just try to briefly explain it in your own words please. People who look at this in the future will need to be able to figure things out.